docs/modules/configure/pages/system.adoc

```diff
@@ -0,0 +1,142 @@
+
+|Organization locales
+|Required
+|You need to configure at least one language. Decidim is multilingual, meaning that you can have most of the admin 
+contents with any number of languages. For seeing other languages you need to configure first the 
+xref:configure:initializer.adoc[Decidim's initializer].
+
+|Force authentication
+|Optional
+|Check if you want to **Force users to authenticate before access organization**. 
+That means visitor they do not see any published content and they only have access to the log-in form.
+
+|Users registration mode
+|Optional
+|Control how you want the login / participant register or signup to work. See below.
+
+|Available authorizations
+|Optional
+|How you want to authorize and verify that your participants are who they say they are. 
+It depends a lot on your Organization, country, etc. See xref:customize:authorizations.adoc[Authorizations customization].
+
+|===
+
+=== Advanced settings
+
+.System - New Organization form advanced settings
+[cols="20h,10h,~"]
+|===
+|Field |Type |Description
+
+|SMTP settings
+|Optional
+|Allows you to configure different SMTP servers for every organization in this installation. 
+See xref:services:smtp.adoc[SMTP configuration].
+
+|Omniauth settings
+|Optional
+|Allows you to configure through external OAUTH providers for every Organization in this installation. 
+See xref:services:social_providers.adoc[Social providers configuration].
+
+|File upload settings
+|Optional
+|Allows you to configure the formats accepted on your instance, as well as default file sizes for file uploads
+
+|Content security policy
+|Optional
+|Allows you to configure organization based rules, but also to configure system wide policies in case your decidim instance is running 
+in multi tenant mode. To learn more, check this https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP[CSP header documentation].
+
+|===
+
+=== Users registration mode
+
+==== Allow participants to register and login
+
+Most of the installations use this setting. Means that everyone can register a participant account and login, without restrictions, only with an email account confirmation (and you can disable that confirmation also if you want to through xref:configure:initializer.adoc[Decidim's initializer])
+
+==== Do not allow participants to register, but allow existing participants to login
+
+This is for when you make a manual import of participants' accounts.
+
+==== Access only can be done with external accounts
+
+In the case where you have an external provider configured (like the `Omniauth settings`) and you want to have accounts from only this provider.
+
+== Edit an organization
+
+Finally, you can also Edit an organization, although some fields cannot be changed as some contents in the database depends on them.
+
+For accessing your newly created Organization, you can do it going to the domain or subdomain that you have configured in `Host`.
```

Comment:
```suggestion
For accessing your newly created Organization, you can do it by going to the domain or subdomain that you have configured in `Host`.
```

Reply:
Should we say something about the confirmation for the admin email and that can be resent in case the SMTP wasn't configured correctly?